xcode/Subconscious/Shared/Services/Noosphere/Noosphere.swift

```diff
@@ -200,13 +200,13 @@ public actor Noosphere {
         .eraseToAnyPublisher()
     }
     
-    func recover(identity: Did, localKeyName: String, mnemonic: RecoveryPhrase) async throws -> Bool {
+    func recover(identity: Did, localKeyName: String, mnemonic: String) async throws -> Bool {
```

Comment:
Update this method to take a string, since that's what all the other Noosphere methods that take a mnemonic take.

This should be fine, since the value of `RecoveryPhrase` is to prevent accidental logging, which isn't a concern in the scope of this method. We would have to unwrap it to a string here anyway.